components/button/_button.scss

```diff
@@ -85,6 +86,30 @@
     }
   }
 
+
+  // Warning button
+  &.sdc-button__warning {
```

Comment:
warning is red? not orange?

Reply:
Hi @einavk !
Do you think is it better to name it danger-button or just red-button?